<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>udg::Q2DViewerExtensionBase</class>
 <widget class="QWidget" name="udg::Q2DViewerExtensionBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>529</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>2</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="m_slicingToolButton" >
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset/>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_windowLevelToolButton" >
         <property name="text" >
          <string>window level</string>
         </property>
         <property name="icon" >
          <iconset/>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_zoomToolButton" >
         <property name="text" >
          <string>zoom</string>
         </property>
         <property name="icon" >
          <iconset/>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_moveToolButton" >
         <property name="text" >
          <string>move</string>
         </property>
         <property name="icon" >
          <iconset/>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_axialViewToolButton" >
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="../main/main.qrc" >:/images/axial.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_sagitalViewToolButton" >
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="../main/main.qrc" >:/images/sagital.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_coronalViewToolButton" >
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="../main/main.qrc" >:/images/coronal.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_rotateClockWiseToolButton" >
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="../main/main.qrc" >:/images/rotateClockWise.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_rotateCounterClockWiseToolButton" >
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="../main/main.qrc" >:/images/rotateCounterClockWise.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_singleViewToolButton" >
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="../main/main.qrc" >:/images/singleView.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_doubleViewToolButton" >
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="../main/main.qrc" >:/images/addViewRight.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_voxelInformationToolButton" >
         <property name="text" >
          <string>voxel information</string>
         </property>
         <property name="icon" >
          <iconset resource="../main/main.qrc" >:/images/voxelInformation.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>42</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>WW/WL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWindowLevelComboBox" name="m_windowLevelComboBox" />
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>42</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_chooseSeriePushButton" >
         <property name="text" >
          <string>Choose Serie To Compare</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_synchroCheckBox" >
         <property name="text" >
          <string>Synchronize Slices</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="m_stackedWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="udg::Q2DViewer" name="m_2DView" />
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="m_viewText" >
           <property name="text" >
            <string>XY : Axial</string>
           </property>
           <property name="scaledContents" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="m_slider" >
           <property name="focusPolicy" >
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="maximum" >
            <number>0</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="m_spinBox" >
           <property name="maximum" >
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="widget" >
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="udg::Q2DViewer" name="m_2DView2_1" />
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="m_viewText2_1" >
               <property name="text" >
                <string>XY : Axial</string>
               </property>
               <property name="scaledContents" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="m_slider2_1" >
               <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="maximum" >
                <number>0</number>
               </property>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="m_spinBox2_1" >
               <property name="maximum" >
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget2" >
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="udg::Q2DViewer" name="m_2DView2_2" />
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="m_viewText2_2" >
               <property name="text" >
                <string>XY : Axial</string>
               </property>
               <property name="scaledContents" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="m_slider2_2" >
               <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="maximum" >
                <number>0</number>
               </property>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="m_spinBox2_2" >
               <property name="maximum" >
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>QWindowLevelComboBox</class>
   <extends>QComboBox</extends>
   <header>qwindowlevelcombobox.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>udg::Q2DViewer</class>
   <extends>QWidget</extends>
   <header>q2dviewer.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../main/main.qrc" />
 </resources>
 <connections/>
</ui>
